homeassistant/components/tuya/sensor.py

```diff
@@ -818,6 +818,27 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             native_unit_of_measurement=UnitOfElectricPotential.VOLT,
             subkey="voltage",
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.CUR_CURRENT,
+            translation_key="current",
```

Comment:
I do not see the strings file updated.

Reply:
It doesn't appear to need an update, there are already keys for current, power & voltage. Also I've tested this on my local system and it didn't have any issue. Am I missing something?

Reply:
These indeed already exists:

![CleanShot 2023-11-25 at 14 56 49@2x](https://github.com/home-assistant/core/assets/195327/8edcd169-c723-4585-8698-d4a56f782b4c)